tests/cpp/null-library-function-cpp-test.yml

```
@@ -0,0 +1,30 @@
+    char buf[128];
+    strcpy(buf, getenv("FOO"));
+    }
+    {
+    fwrite("foo", 3, 1, fopen("foo.txt", "w"));
+    }
+    {
+    FILE *fptr;
+    fwrite("foo", 3, 1, fptr = fopen("foo.txt", "w"));
+    }
+    void test_getc() {
+    int c = getc(fopen(file_name, "r")); 
```

yamllint

[error] trailing spaces

(trailing-spaces)